recipes/wasmtime/all/conandata.yml

```diff
@@ -1,4 +1,34 @@
 sources:
+  "25.0.2":
+    Windows:
+      "x86_64":
+        url: "https://github.com/bytecodealliance/wasmtime/releases/download/v25.0.2/wasmtime-v25.0.2-x86_64-windows-c-api.zip"
+        sha256: "fdd457be3dd3e0a7c923f92fe91dd1b892f15450a64cd17ad8433eb097c6988e"
+    MinGW:
+      "x86_64":
+        url: "https://github.com/bytecodealliance/wasmtime/releases/download/v25.0.2/wasmtime-v25.0.2-x86_64-mingw-c-api.zip"
+        sha256: "8385665e53340ca3c5c883e4f2874b6c5564a592ed3cbe4f1a6d077e416df2ab"
+    Linux:
+      "x86_64":
+        url: "https://github.com/bytecodealliance/wasmtime/releases/download/v25.0.2/wasmtime-v25.0.2-x86_64-linux-c-api.tar.xz"
+        sha256: "73aade8ed52a07bbc07c4286f1249e82f7b2d9764e97dc4b7c1c5be02b354ee7"
+      "armv8":
+        url: "https://github.com/bytecodealliance/wasmtime/releases/download/v25.0.2/wasmtime-v25.0.2-aarch64-linux-c-api.tar.xz"
+        sha256: "678bf7212f4f97e7b751b0630e0fb6b2e3f9169bc12f7732b83a4b0a0403d596"
+      "s390x":
+        url: "https://github.com/bytecodealliance/wasmtime/releases/download/v25.0.2/wasmtime-v25.0.2-s390x-linux-c-api.tar.xz"
+        sha256: "91be4ca268830798d29704b5f32521c70ba3aa1a1d4d078c96b36105f76e7bbe"
+    Macos:
+      "x86_64":
+        url: "https://github.com/bytecodealliance/wasmtime/releases/download/v25.0.2/wasmtime-v25.0.2-x86_64-macos-c-api.tar.xz"
+        sha256: "a0b33054fd3b5cf9707d1a39a68a80d11600e2fcdf6074ba4263b9c339bfe64b"
+      "armv8":
+        url: "https://github.com/bytecodealliance/wasmtime/releases/download/v25.0.2/wasmtime-v25.0.2-aarch64-macos-c-api.tar.xz"
+        sha256: "8cc68c43559c78e103bebf63ba16b768f7bb9c0ba164649279c1a09be690b416"
+    Android:
+      "armv8":
+        url: "https://github.com/bytecodealliance/wasmtime/releases/download/v25.0.2/wasmtime-v25.0.2-aarch64-linux-c-api.tar.xz"
```

Comment:
This is the linux release, but there seems to be a typo? The release contains Android files. Is this duplicatin in pourpose?